### PR TITLE
Add 2FAuth to Misc/Other

### PR DESCRIPTION
**Pull request by Bubka**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements (`Demo` and `Clients` are optional, don't add a duplicate `Source code` link if it is the same as the main link):

``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and marked `⊘ Proprietary`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks/specifc dialects.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is "Add aaa to bbb" for adding software aaa to section bbb. Do not call it "Update README.md".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -998,6 +998,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 
 **[`^        back to top        ^`](#)**
 
+- [2FAuth](https://github.com/Bubka/2FAuth) - A web app to manage your Two-Factor Authentication (2FA) accounts and generate their security codes. ([Demo](https://demo.2fauth.app/)) `AGPL-3.0` `PHP`
```
Comment (n8225) [inline README.md:1001]: Wouldn't this be a better fit in the Password Managers category?

**Review by n8225 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Bubka) [reply README.md:1001]: This was my first thought, but all items of the PM category are strictly password managers unlike 2FAuth.

Both relates to the same purpose, manage security artifacts of accounts. Therefore shouldn't the category be renamed to something a little less restrictive ? i.e _Account & Password managers_ ?

Comment (nodiscc) [reply README.md:1001]: I think it should rather go to https://github.com/n1trux/awesome-sysadmin#identity-management

Comment (Bubka) [reply README.md:1001]: But 2FAuth is not a sysadmin tool, it's exactly a selfhosted app and it is why I started to develop it, to have a selfhosted solution for a daily end-user need, the 2FA token generation.

The target audience is clearly the one of the awesome-selfhosted page in my opinion.

**Comment by Bubka** [comment]
Hello,

What about my PR so? Should I changed the category to _Password managers_ or _Misc/Other_ is acceptable ? Any other opinions ?

**Review by nodiscc — APPROVED** [review]

**Pull request merged**